components/mpu60xx/idf_component.yml

```
@@ -0,0 +1,27 @@
+## IDF Component Manager Manifest File
+
+description: MPU60xx I2C driver
+ 
+maintainers: 
+  - "Rohan Jeet <jeetrohan92@gmail.com>"
+
+dependencies:
+  ## Required IDF version
+  idf:
+    version: ">=4.1.0"
```

the new I2C driver was added in v5.2
```suggestion
    version: ">=5.2.0"
```

---

This was the default from the idf.py and I wasn't mindful even though I thought about looking into it earlier. 
I'd be thankful if you'll update this for me.

---

If you're new to ESP-IDF, understanding the various releases can be challenging. The [documentation](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/versions.html) provides an overview if you'd like to learn more.

Generally, we recommend starting with the latest stable release and only upgrading to bugfix releases, unless a specific new feature is required.

However, for driver development,you should specify which ESP-IDF versions your driver supports. The latest I2C driver update is detailed in the "Major Features" section of [v5.2 release](https://github.com/espressif/esp-idf/releases/tag/v5.2). The team is also working on improving how new features and changes are presented.